src/index.js

```diff
@@ -79,9 +81,16 @@ app.use(async (msg) => {
       return;
     }
   }
+
   const ctx = { ...GLOBAL_CTX, msg };
   const response = await commandHandler(ctx, arg);
-  msg.channel.send(response);
+  GLOBAL_CTX.player = ctx.player;
+  GLOBAL_CTX.enemy = ctx.enemy;
+  GLOBAL_CTX.inventory = ctx.inventory;
```

Comment:
Forgot about this last time, we have to update the `GLOBAL_CTX` after the command handler is run, since modifications to `ctx`, such as `ctx.player = new Player`, will not be reflected in `GLOBAL_CTX` otherwise.